deployments/production/ml-sots-cronjob.json

```diff
@@ -0,0 +1,174 @@
+                  {
+                    "name": "DATABASE_URL",
+                    "valueFrom": {
+                      "secretKeyRef": {
+                        "key": "databaseurl",
+                        "name": "ml-sots-secrets"
+                      }
+                    }
+                  },
+                  {
+                    "name": "DATABASE_USER",
+                    "valueFrom": {
+                      "secretKeyRef": {
+                        "key": "databaseuser",
+                        "name": "ml-sots-secrets"
+                      }
+                    }
+                  },
+                  {
+                    "name": "DATABASE_PASSWORD",
+                    "valueFrom": {
+                      "secretKeyRef": {
+                        "key": "databasepass",
+                        "name": "ml-sots-secrets"
+                      }
+                    }
+                  },
+                  {
+                    "name": "CONCIERGE_DATABASE_URL",
+                    "valueFrom": {
+                      "secretKeyRef": {
+                        "key": "concierge_databaseurl",
+                        "name": "ml-sots-secrets"
+                      }
+                    }
+                  },
+                  {
+                    "name": "CONCIERGE_DATABASE_USER",
+                    "valueFrom": {
+                      "secretKeyRef": {
+                        "key": "concierge_databaseuser",
+                        "name": "ml-sots-secrets"
+                      }
+                    }
+                  },
+                  {
+                    "name": "CONCIERGE_DATABASE_PASSWORD",
+                    "valueFrom": {
+                      "secretKeyRef": {
+                        "key": "concierge_databasepassword",
+                        "name": "ml-sots-secrets"
+                      }
+                    }
+                  },
+                  {
+                    "name": "TAXO_HOST",
+                    "value": "taxonomer.wink.com"
+                  }
+                ],
+                "image": "quay.io/winkapp/ml-sots:production",
+                "imagePullPolicy": "Always",
+                "name": "ml-sots",
+                "ports": [
+                  {
+                    "containerPort": 4040,
+                    "protocol": "TCP"
+                  }
+                ],
+                "resources": {
+                  "limits": {
+                    "cpu": "1500m",
+                    "memory": "14000Mi"
+                  },
+                  "requests": {
+                    "cpu": "1000m",
+                    "memory": "11000Mi"
+                  }
+                }
+              }
+            ],
+            "imagePullSecrets": [
+              {
+                "name": "winkapp-ml-sot-deployment-pull-secret"
+              }
+            ],
+            "nodeSelector": {
+              "instance-type": "memory"
```

Comment:
This is the one we want on the autoscale-exempt node. Doesn't that need the "sot" instance-type?